src/tools/crossgen2/Common/TypeSystem/Common/TypeHashingAlgorithms.cs

```diff
@@ -251,5 +251,38 @@ public static int ComputeSignatureVariableHashCode(int index, bool method)
                 return index * 0x5498341 + 0x832424;
             }
         }
+
+        private static uint XXHash32_MixEmptyState()
```

Comment:
These should probably go into ReadyToRunHashCode.cs.

TypeHashingAlgorithms.cs is hashing used within the type system. Crossgen2 is one of the consumers of the type system, but we have many more (CoreRT compiler, CoreRT type loader, ILVerify, potentially other things in the future) and these methods don't make sense there.

Reply:
Working on it

Reply:
Fixed